src/mac.rs

```diff
@@ -58,10 +59,10 @@ impl ProjectDirectories {
     }
 
     pub fn from_project_name(project_name: &str) -> ProjectDirectories {
-        from_unprocessed_string(name)
+        ProjectDirectories::from_unprocessed_string(project_name.to_string())
```

Comment:
Is the `to_string()` necessary here?

Looking at e. g. https://github.com/soc/directories-rs/blob/master/src/win.rs#L72 the same code works without it.

Reply:
You're calling `from_unprocessed_string` which expects a String. I have not looked into code any further :)

Reply:
Oh, on Linux, the first thing `from_unprocessed_string` does is `let project_name = String::from(value);`, so it's also allocating a String there.

You could probably use `Cow<str>` but I don't think it's a huge performance loss.